docs/en/manuals/live-update.md

```diff
@@ -127,218 +61,46 @@ When bundling, any excluded resource will be left out of the application bundle.
+The .zip file also contains a manifest file that describes the meta data for each resource within the .zip file.
 
-```lua
-...
-elseif message_id == hash("attempt_download_manifest") then
-    local base_url = "https://my-game-bucket.s3.amazonaws.com/my-resources/" -- <1>
-    http.request(base_url .. MANIFEST_FILENAME, "GET", function(self, id, response)
-        if response.status == 200 or response.status == 304 then
-            -- We got the response ok.
-            print("verifying and storing manifest " .. MANIFEST_FILENAME)
-            resource.store_manifest(response.response, store_manifest_cb) -- <2>
-        else
-            -- ERROR! Failed to download manifest!
-            print("Failed to download manifest: " .. MANIFEST_FILENAME)
-        end
-    end)
-end
-```
-1. The manifest will be stored on S3 next to the rest of the Live update resources. As before, if you create a Zip archive with resources you need to host the files somewhere and reference their location when downloading them with `http.request()`.
-2. Similar to how resources are downloaded and stored, the call to `resource.store_manifest` takes the downloaded manifest data and a callback as arguments. The function will verify the manifest and persist it to local storage.
+While our current pipeline only supports creating a single .zip file, it is in fact possible to split that zip file into smaller .zip files. This allows for smaller downloads for a game: level packs, seasonal content etc.
 
-If `resource.store_manifest` succeeds, the new manifest is now located in local storage. The next time the engine starts this manifest will be used instead of the one bundled with the game.
+## Content verification
 
-### Caveats
-There are a few gotchas that might be good to know if you plan to use this feature to store a new manifest with Live update.
+One of the major features of the our live update system, is that you can use now use many content archives, potentially from many different Defold versions.
 
-* It is only possible to add or modify resources referenced by collection proxies that are tagged as `Exclude` in the new manifest. No changes should be made to already bundled resources or resources not in excluded collection proxies. For example, doing changes in a script that is referenced by a bundled collection will cause the resource system to look for that resource in the bundled data archive. But since the shipped game bundle has not changed (only the manifest has) the changed script will not be found and consequently cannot be loaded.
+The `liveupdate.add_mount()` default behavior, is to add an engine version check when attaching a mount.
+This means that both the game base archive and liveupdate archive(s) need to be created at the same time with the same engine version. This will invalidate any previously downloaded archives by the client, forcing them to redownload the content.
```

Comment:
At the same time? Is there a timestamp? Or is it "with the same engine version" which matters?

Reply:
I've noticed that we use "live update" and "liveupdate" in different places in the docs (not only these).

I think we should consistently write "live update", possibly even "Live Update".